firefox-ios/firefox-ios-tests/Tests/XCUITests/DesktopModeTests.swift

```diff
@@ -41,9 +41,17 @@ class DesktopModeTestsIpad: IpadOnlyTestCase {
     }
 }
 
-class DesktopModeTestsIphone: IphoneOnlyTestCase {
+class DesktopModeTestsIphone: FeatureFlaggedTestBase {
+    override func setUp() {
```

Comment:
I went with this solution since I can't have `IphoneOnlyTestCase` and `FeatureFlaggedTestBase` inheritance at the same time